src/cloudstate/channel_state.ts

```diff
@@ -13,7 +13,8 @@ export class ChannelState {
   get() {
     return this.messages.map(m=>m.get());
   }
-  post(message:Message) {
+  post(text: string) {
```

Comment:
we don't support passing classes from client to server right now. the data must be serializable.

Reply:
this is why m.get didn't work. the array of messages was actually an array of {text: string} not an array of Message instances because the class was serialized to just a plain object. we may support this in the near future or we may add clearer errors to disallow this.